### ISSUE TITLE
A good set of tags to start with ?

### ISSUE THREAD

**Issue by valerianrossigneux** [vertexproject/synapse#812]
Hi, is there somewhere a good set of tags/analytical observation to start with ?

**Comment by valerianrossigneux** [comment]
Apologies for not making it clear, but yes it is for threat intelligence.

> With time and experience a "stable" set of tags will settle out.
The categories you highlighted are a good starting point. But essentially what you bare saying is it would be different for every org/intel analyst. I wish there was a comprehensive list of these hierarchic tags coming from experts best practices, lots of them are anyway very common for everybody I think.  Flexibility is great but a baseline set of tags would allow people to have an easier starting point as well as share with others on a common set of tags.

**Comment by therealsilence** [comment]
I agree that shared "best practices" are helpful - if others can leverage your experience, it may save people having to reinvent the wheel. With respect to Synapse, I think a few things may help:
- We will be updating the Synapse documentation in the near future, and ideally would like to include expanded tutorials to show people some of the ways Synapse can be used. This will help highlight some of the tags / tag hierarchies and how they can be used.
- The tags are tightly tied to the data model. A tag has a specific meaning; a tag applied to a particular node / form in the model also has a very specific meaning. So to some extent agreeing on tags (to the extent that's desirable) also implies agreement on how the tags should be used (where / when they should be applied). This impacts "best practices" to the extent that different users may leverage some parts of the model more than others, depending on their focus.
- While we want to show users how tags *can* be used, I do believe in the power and flexibility that allows analysts to create and adapt tags on the fly (though ideally within some sort of oversight - see https://vertexprojectsynapse.readthedocs.io/en/latest/synapse/userguides/ug009_dm_tagmodel.html#tag-considerations for some thoughts on this). It's common for analysts to identify new observations in the course of their work - allowing them to record that as it occurs (as opposed to waiting for a newly proposed tag to be approved) is very powerful.
- In the grand scheme of things, I would love to see wide adoption of Synapse as an analytical framework, whether for threat intelligence, financial analysis, or any other field. The hope is that as more people doing similar analysis adopt the same platform, that drives a desire to share and exchange information. For people or organizations sharing data, there is an incentive to standardize, which includes standardized tags. So hopefully "accepted best practices" for various fields will gel out of that (a process I would prefer to some working group sitting in endless committee to make those decisions).

**Comment by valerianrossigneux** [comment]
Thanks a lot, I really appreciate the thorough answer!

**Comment by therealsilence** [comment]
Any time!
